include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3527,6 +3533,7 @@ inline FEEvaluationBaseData<dim, Number, is_face, VectorizedArrayType>::
   , quadrature_weights(descriptor->quadrature_weights.begin())
   , cell(numbers::invalid_unsigned_int)
   , is_interior_face(is_interior_face)
+  , is_cell_neighbor(false)
```

Comment:
Since, they are next to each other: can not we misuse `is_interior_face`?

Reply:
We could since `is_interior_face` is only used for faces but the logic would look confusing in the if statements IMO.

Reply:
I would simply rename it. We use common function/variables all over the base classes.

Reply:
I agree, we could choose a more generic name here, if the logic works with a single variable.

Reply:
Do you think of something like `is_interior_face_or_cell_neighbor`?

Reply:
Yes and no. I would revert the logic, since `interior face`=`this cell` and `exterior face`=`cell neighbor`.

Reply:
WDYT? @peterrum Would you prefer to inline the logic?